packages/backend/src/build-disk-image.ts

```diff
@@ -168,7 +173,11 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
 
         // Step 3.1 Since we have started the container, we can now go get the logs
         await logContainer(build.engineId, containerId, progress, data => {
-          logData += data;
+          try {
+            fs.appendFileSync(logPath, data);
+          } catch (e) {
+            // ignore
```

Comment:
would be nice to log as console.debug in case you need to know why it's failing

Reply:
Fixed.